components.html

```diff
@@ -0,0 +1,1381 @@
+
+
+
+      <!--BUTTONS-->
+
+      <div class="flex-container">
+
+        <!--Primary button-->
+        <button class="btn">Label</button> &nbsp;&nbsp;
+
+        <!--Stroke button-->
+        <button class="btn btn-stroke">Label</button> &nbsp;&nbsp;
+
+        <!--Disabled button-->
+        <button class="btn btn-disabled">Label</button> &nbsp;&nbsp;
+
+        <!--Primary button (Small)-->
+        <button class="btn btn-sm">Label</button> &nbsp;&nbsp;
+
+        <!--Stroke button (Small)-->
+        <button class="btn btn-sm btn-stroke">Label</button> &nbsp;&nbsp;
+
+        <!--Disabled button (Small)-->
+        <button class="btn btn-sm btn-disabled">Label</button> &nbsp;&nbsp;
+
+        <!--Icon button (Small)-->
+        <button class="btn btn-sm btn-stroke">
+          <div class="flex-container">
+            <div class="btn-sm-icon">
+              <svg class="icon-16 btn-tooltip-icon">
+                <use xlink:href="img/icons.svg#add" />
+              </svg>
+            </div>
+            Label
+        </button> &nbsp;&nbsp;
+
+        <!--Icon button (Toolip)-->
+        <button class="btn btn-tooltip">
+          <div class="flex-container">
+            <div>
+
+              <svg class="icon-16 btn-tooltip-icon">
+                <use xlink:href="img/icons.svg#check-white" />
+              </svg>
+
+            </div>
+            Save
+        </button>
+
+        <!--Icon button (Toolip outline)-->
+        <button class="btn-tooltip-stroke">Cancel</button>
+
+      </div>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Action Controls</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Default Action-->
+      <div class="flex-container">
+        <button class="flex-container action-icon btn-no-outline">
+          <p class="action-title text-center">Action Title</p>
+          <svg class="icon-24">
+            <use xlink:href="img/icons.svg#action-backup-database" />
+          </svg>
+        </button>
+
+        <!--Locked Action-->
+        <button class="flex-container action-icon btn-no-outline">
+          <p class="action-title-locked text-center">Action Title</p>
+          <svg class="icon-24">
+            <use xlink:href="img/icons.svg#action-locked" />
+          </svg>
+        </button>
+      </div>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Checkbox / Radio Button / Toggle Switch</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Checkbox-->
+      <ul>
+        <li>
+          <input type="checkbox" class="checkbox-default" name="" value="">
+          <label>Checkbox label</label>
+        </li>
+      </ul>
+
+
+
+      <!--Radio Button-->
+      <ul>
+        <li>
+          <input type="radio" name="name" value="">
+          <label>Radio button label</label>
+        </li>
+      </ul>
+
+
+
+      <br />
+
+
+
+      <!--Toggle-->
+      <div class="switch_box box_1">
+        <input type="checkbox" class="switch_1">
```

Comment:
using both underscores and dashes for classnames, should probably standardize on one or the other.